### PR TITLE
Added gentlentapis.com

### PR DESCRIPTION
**Pull request by tomkln**
* [x] Description of Organization
* [x] Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

Description of Organization
====

Organization Website: https://www.gentlent.com

I am CEO/CTO at Gentlent, which is a software development and consulting company.

Reason for PSL Inclusion
====

gentlentapis.com will be used to host multiple (incl. third-party) APIs on different subdomains which should not be able to set cookies to other subdomains for security reasons. User content may be allowed. Browsers should, therefore, recognize gentlentapis.com as a suffix.

DNS Verification via dig
=======

```
dig +short TXT _psl.gentlentapis.com
"https://github.com/publicsuffix/list/pull/984"
```

make test
=========

"make test" ran successfully: https://gist.github.com/tomkln/e6d5f0a747a3b2433ab2db2b0abe7428

### CODE REVIEW

File: public_suffix_list.dat [hunk 1]
```diff
@@ -11797,7 +11797,7 @@ usercontent.jp
 // Gentlent, Inc. : https://www.gentlent.com
 // Submitted by Tom Klein <tom@gentlent.com>
 gentapps.com
-lab.ms
```
Comment (dnsguru) [inline public_suffix_list.dat:11800]: Removal of entries, even coupled within a header section like this requires the domains be validated.

We'll need the removal of lab.ms to be submitted with DNS authority as a separate PR

Please revise this back in

**Comment by tomkln** [comment]
Thanks for the change, then I'll create a separate PR for this entry soon.

File: public_suffix_list.dat (hunk 1 above)
Comment (tomkln) [reply public_suffix_list.dat:11800]: Change commited
